src/memory.cpp

```diff
@@ -14,6 +14,7 @@
 
 #include <cmath>
 
+#include "CL/cl.h"
```

Comment:
This is already pulled via `memory.hpp`. The OpenCL headers are included via `cl_headers.hpp`.

Reply:
Sorry, I think it has been added automatically by clangd and I forgot to remove it